runtime/etc/lighttpd/lighttpd.conf

```diff
@@ -7,12 +7,4 @@ server.errorlog = "/dev/fd/2"
 accesslog.format = env.LIGHTTPD_ACCESSLOG_FORMAT
 accesslog.filename = "/dev/fd/2"
 
-fastcgi.server = ( "" =>
-  (
-	(
-	  "host" => env.LIGHTTPD_FASTCGI_HOST,
-	  "port" => env.LIGHTTPD_FASTCGI_PORT,
-	  "check-local" => "disable",
-	),
-  )
-)
+include_shell "./gen-server"
```

Comment:
My prior comment is not resolved.  Your startup script `/usr/local/bin/start-server` does not `cd /etc/lighttpd`, so this is another case of "you have not tested well".

Reply:
I tested it again, and it is working locally and in the container :-)
It looks like that lighttpd changed the working directory!

Reply:
Are you sure that you're testing lighttpd?  It appears that you're testing Apache.
`SERVER=apache` in the Dockerfile.

Reply:
Yes, the variable is overridden here:
https://github.com/camptocamp/docker-mapserver/blob/master/acceptance_tests/docker-compose.yaml#L72
and here:
https://github.com/camptocamp/docker-mapserver/blob/master/acceptance_tests/docker-compose.yaml#L87
:-)